owners/index.js

```diff
@@ -14,40 +14,16 @@
  * limitations under the License.
  */
 
-require('dotenv').config();
```

Comment:
Sanity check whenever a `require` statement is deleted: Should the package be removed from `package.json`?

Reply:
This isn't removing the actual require. `require('dotenv').config()` is the canonical way to initialize reading from a `.env` file, but that has moved into `bootstrap.js`. So, tldr, nothing need be removed from `package.json`